core/0.18.0/docker-entrypoint.sh

```diff
@@ -15,6 +15,14 @@ if [[ "$1" == "bitcoin-cli" || "$1" == "bitcoin-tx" || "$1" == "bitcoind" || "$1
 		CONFIG_PREFIX=$'mainnet=1\n[main]'
 	fi
```

Comment:
We should use `elif` instead with `else` which reset `BITCOIN_NETWORK` to empty string if it does not match anything.